packages/istanbul-reports/lib/html-spa/assets/spa.css

```diff
@@ -74,183 +129,171 @@ a:hover {
+}
+
+/* Toggle */
+
+.toggle {
+    display: inline-flex;
+    align-items: center;
 }
 
+.toggle__label {
+    margin-right: 0.5em;
+}
+
+.toggle__options {
+    display: inline-block;
+    border: 1px solid #0074d9;
+    border-radius: 4px;
+    color: #0074d9;
+    overflow: hidden;
+    white-space: nowrap;
+}
+
+.toggle__option {
+    padding: 4px 8px;
+    background: #fcfcfc;
+}
+
+.toggle__option + .toggle__option {
+    border-left: 1px solid #0074d9;
+}
+
+.toggle__option.is-toggled {
+    color: #fff;
+    background: #0074d9;
+    border-left-color: #fcfcfc;
+}
+
+/* Expand */
+
+.expandbutton {
+    display: inline-block;
+    width: 1em;
+    margin-right: 0.25em;
+    padding: 0;
+    background-color: transparent;
+    font-weight: bold;
+}
+
+/* Fraction */
+
 .fraction {
-    font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
-    font-size: 10px;
-    color: #555;
-    background: #e8e8e8;
-    padding: 4px 5px;
-    border-radius: 3px;
-    vertical-align: middle;
+    font-size: 12px;
+    color: #666;
+    padding: 2px 4px;
+    border-radius: 4px;
 }
 
+/* Coverage */
+
 .coverage-summary {
     border-collapse: collapse;
-    border: 1px solid #bbb;
-    width: 100%;
-}
-.coverage-summary tr {
-    border-bottom: 1px solid #bbb;
 }
-.coverage-summary td,
-.coverage-summary th {
-    padding: 5px 7px;
+
+.coverage-summary tbody tr {
+    border-bottom: 1px solid #fff;
 }
+
 .coverage-summary td,
 .coverage-summary th {
-    border-right: 1px solid #bbb;
-}
-.coverage-summary td:last-child,
-.coverage-summary th:last-child {
-    border-right: none;
+    padding: 5px;
 }
+
 .coverage-summary th {
-    text-align: left;
+    text-align: center;
     font-weight: normal;
     white-space: nowrap;
 }
-.coverage-summary th.pic,
+
 .coverage-summary th.abs,
 .coverage-summary td.pct,
 .coverage-summary td.abs {
     text-align: right;
 }
+
+.coverage-summary th.file {
+    min-width: 300px;
+    text-align: left;
+}
+
 .coverage-summary td.file {
     white-space: nowrap;
 }
-.coverage-summary td.pic {
-    min-width: 120px !important;
-}
-.coverage-summary .sorter {
-    height: 10px;
-    width: 7px;
-    display: inline-block;
-    margin-left: 0.5em;
-    background: url(sort-arrow-sprite.png) no-repeat scroll 0 0 transparent;
-}
-.coverage-summary .sorted .sorter {
-    background-position: 0 -20px;
-}
-.coverage-summary .sorted-desc .sorter {
-    background-position: 0 -10px;
-}
-/* dark red */
-.red.solid,
-.low .cover-fill {
-    background: #c21f39;
-}
-/* light red */
-.low {
-    background: #fce1e5;
-}
-/* light green */
-.high {
-    background: rgb(230, 245, 208);
-}
-/* dark green */
-.high .cover-fill {
-    background: rgb(77, 146, 33);
-}
-/* dark yellow (gold) */
-.medium .cover-fill {
-    background: #f9cd0b;
+
+.coverage-summary td.pct {
+    font-weight: 400;
 }
-/* light yellow */
-.medium {
-    background: #fff4c2;
+
+.coverage-summary td.abs {
+    color: #666;
+    font-weight: 200;
```

Comment:
This degrades the view, on my system it causes the render of these numbers to appear slightly pixelated.  I think leaving `font-weight` as is (normal) with the `font-size: 12px` adequately de-emphasizes the covered/total absolute counts while still allowing a smooth looking font render.  Tested by disabling this rule with the browser devtools.

Reply:
Fair point -- the fancy font weight was perhaps gilding the lily a bit ;)

I've removed the font weight on my branch now, so hopefully you can merge now.